redshift/aws_lambda/lambda_function.py

```diff
@@ -50,10 +50,10 @@ def lambda_handler(request, context):
     # Make sure you should know these details about the table you are migrating beforehand
     # Set the "limit" according to your estimates of the table's size and row count
     # Again, these can also be stored in "request"
-    table = request["table"]
-    primary_key = request["primary_key"]
-    cursor = request["cursor"]
-    limit = request["limit"]
+    table = request['secrets']["table"]
```

Comment:
Single-quote/double-quote inconsistency. The Google Python style guide is impartial to both, but strongly recommends you pick one and stick to it. 

I prefer single quotes for almost everything, except in the case where they should be taken literally, like in the SQL query below.